dev/packages/beats/activemq-0.0.1/dataset/audit/agent/stream/log.yml.hbs

```diff
@@ -0,0 +1,6 @@
+type: log
+paths:
+{{#each paths as |path i|}}
```

Comment:
Is the second part `as |path i|` needed?

Reply:
This is tricky actually, because the "i" is always present when unrolling the "range". Wouldn't like to detect this in the code. I will let the developer adjust it later on. Is it fine?

Reply:
SGTM as long as this works on the Kibana side (I assume it does).